Model/Import/Attributes.php

```diff
@@ -2,60 +2,205 @@
 
 namespace SITC\Sinchimport\Model\Import;
 
+/**
+ * Class Attributes
+ * @package SITC\Sinchimport\Model\Import
+ */
 class Attributes {
 
+    /**
```

Comment:
Commenting things is all well and good, but when it gets to the point that you're commenting somebody else's code and the comment is basically identical to the package/class/variable/const/etc's name, it's probably excessive